SDK_versioned_docs/version-3.0.0/subgraph-example.md

```diff
@@ -0,0 +1,282 @@
+---
+id: Subgraph examples
+sidebar_position: 3
+---
+
+# Subgraph Query Examples
+You can use the subgraph to make GraphQL queries to fetch data about Uniswap V3. You can fetch data points like the spot price of a token, current liquidity of a pool, volume on a certain day, and much more. Below are some example queries. You can copy and paste them in the [v3 explorer](https://thegraph.com/hosted-service/subgraph/uniswap/uniswap-v3) to get fresh data.
+
+## Global Data
+To query global data, pass in the Uniswap Factory address : `0x1F98431c8aD98523631AE4a59f267346ea31F984` and select the desired fields.  Reference the full [factory schema](https://github.com/Uniswap/v3-subgraph/blob/main/schema.graphql#L1) to see all possible fields.
+
+### Current Global Data
+An example querying total pool count, transaction count, and total volume in USD and ETH:
+```
+{
+  factory(id: "0x1F98431c8aD98523631AE4a59f267346ea31F984" ) {
+    poolCount
+    txCount
+    totalVolumeUSD
+    totalVolumeETH
+  }
+}
+```
+
+### Historical Global Data
+You can also query historical data by specifying a block number.
+```
+{
+  factory(id: "0x1F98431c8aD98523631AE4a59f267346ea31F984", block: {number: 13380584}){
+    poolCount
+    txCount
+    totalVolumeUSD
+    totalVolumeETH
+  }
+}
+```
+
+## Pool Data
+To get data about a certain pool, pass in the pool address. Reference the full [pool schema](https://github.com/Uniswap/v3-subgraph/blob/main/schema.graphql#L75) and adjust the query fields to retrieve the data points you want.
+
+### General Pool Query
+The query below returns the feeTier, spot price, and liquidity for the ETH-USDC pool.
+
+```{
+  pool(id: "0x8ad599c3a0ff1de082011efddc58f1908eb6e6d8") {
+    tick
+    token0 {
+      symbol
+      id
+      decimals
+    }
+    token1 {
+      symbol
+      id
+      decimals
+    }
+    feeTier
+    sqrtPrice
+    liquidity
+  }
+}
+```
+
+### All Possible Pools
+The maxiumum items you can query at once is 1000. Thus to get all possible pools, you can interate using the skip variable. To get pools beyond the first 1000 you can also set the skip as shown below.
```

Comment:
'maximum' typo.  Also is the 1000 limit imposed by the Graph? always helpful to explain why limitations exist

Reply:
and then you 'must' iterate right?